.gitignore

```diff
@@ -20,3 +20,4 @@ pip-wheel-metadata
 .vscode
 Pipfile
 Pipfile.lock
+venv/
```

Comment:
I don't think there's any likely harm in essentially globally reserving "venv" throughout the tree since nobody's likely to use it casually; but maybe this should be anchored to the base directory as `/venv/` just in case? (This could probably apply to most of the others too.) I did that, for example, with `/.env`, but didn't try to retrofit the other entries.

And we _do_ already have `.venv` and we probably don't want to spawn too many variations.

Reply:
Most of them I suspect should be anchored, but there's always the chance that some will occur at lower levels of the directory tree. Leaving them unanchored is "flexible" but risks "real files" being ignored by git unintentionally. I'm not sure that's a realistic concern with any of the names we're excluding at this point, but going forward I'd prefer that we try to make new patterns as specific as appropriate rather than just leave them unanchored because it's easy.

Reply:
I anchored the ones which were obvious from my checkout.

Reply:
I opted instead to change my work environment rather than changing this file, so I've rescinded these changes.